python-sdk/src/astro/sql/operators/export_file.py

```diff
@@ -62,6 +73,71 @@ def execute(self, context: Context) -> File:
         else:
             raise FileExistsError(f"{self.output_file.path} file already exists.")
 
+    def get_openlineage_facets(self, task_instance) -> OpenLineageFacets:  # skipcq: PYL-W0613
+        """
+        Collect the input, output, job and run facets for export file operator
+        """
+        input_dataset: list[OpenlineageDataset] = []
+        if isinstance(self.input_data, BaseTable):
```

Comment:
@utkarsharma2 do we need check for temp table config here https://github.com/astronomer/astro-sdk/pull/1144/files#r1012835723

Reply:
added condition for temp table